package.json

```diff
@@ -9,6 +9,7 @@
   "author": "Chris ter Beke <chris@printr.com>",
   "license": "GPL-2.0",
   "scripts": {
+    "start": "NODE_ENV=development node app",
```

Comment:
This can be removed, the new client has mores specific cases. `npm start` is also not used by us anywhere.